entropy/entropy.go

```diff
@@ -26,3 +30,46 @@ func GetRandom(source EntropySource, len uint32) ([]byte, error) {
 	}
 	return randomBytes, nil
 }
+
+// EntropyReader hold info for user entropy to be given to the dkg
+// Warning: UserOnly should be set to true for debugging, and randomness coming from user
+// should always be mixed with crypto/rand and thus userOnly dhoul be false by default
+type EntropyReader struct {
+	Entropy  string
+	UserOnly bool
+}
+
+// Read calls the executable as many times needed to fill the array p
+// n == len(p) if and only if err == nil
+func (r *EntropyReader) Read(p []byte) (n int, err error) {
+	if r.Entropy == "" {
+		return 0, errors.New("No reader was provided")
+	}
+	var b bytes.Buffer
+	read := 0
+	for read < len(p) {
+		cmd := exec.Command("./" + r.Entropy)
```

Comment:
Make that `r.Entropy` directly the path of the executable without the `./` in front, it should work